src/src/pages/capabilities/SelectedCapabilityContext.js

```diff
@@ -90,6 +94,13 @@ function SelectedCapabilityProvider({ children }) {
     kafkaClusterTopicList();
   }, [clustersList]);
 
+  // load bypass rights:
+  const getBypassMembershipApplication = useCallback(async () => {
```

Comment:
On second read-through; Why don't we utilize our generic approach to sending requests to the server and updating a variable upon completion. That is exactly what you are doing here, is it not?

The `useSelfServiceRequest` should suffice?